File: system-test/java/src/test/java/com/mycompany/myshop/systemtest/legacy/mod02/smoke/system/MyShopUiSmokeTest.java
package com.mycompany.myshop.systemtest.legacy.mod02.smoke.system;

import com.mycompany.myshop.systemtest.legacy.mod02.base.BaseRawTest;
import org.junit.jupiter.api.BeforeEach;
import org.junit.jupiter.api.Test;

import static org.assertj.core.api.Assertions.assertThat;

class MyShopUiSmokeTest extends BaseRawTest {
    @BeforeEach
    void setUp() {
        setUpMyShopBrowser();
    }

    @Test
    void shouldBeAbleToGoToMyShop() {
        var response = myShopUiPage.navigate(getMyShopUiBaseUrl());

        assertThat(response.status()).isEqualTo(200);

        var contentType = response.headers().get("content-type");
        assertThat(contentType).isNotNull().contains("text/html");

        var pageContent = myShopUiPage.content();
        assertThat(pageContent).contains("<html").contains("</html>");
    }
}

File: system-test/dotnet/SystemTests/Legacy/Mod02/SmokeTests/System/MyShopUiSmokeTest.cs
using SystemTests.Legacy.Mod02.Base;
using Xunit;

namespace SystemTests.Legacy.Mod02.SmokeTests.System;

public class MyShopUiSmokeTest : BaseRawTest
{
    public override async Task InitializeAsync()
    {
        await base.InitializeAsync();
        await SetUpMyShopBrowserAsync();
    }

    [Fact]
    public async Task ShouldBeAbleToGoToMyShop()
    {
        var response = await shopUiPage!.GotoAsync(_configuration.MyShopUiBaseUrl);

        Assert.Equal(200, response!.Status);

        var contentType = response.Headers["content-type"];
        Assert.NotNull(contentType);
        Assert.Contains("text/html", contentType);

        var pageContent = await shopUiPage.ContentAsync();
        Assert.Contains("<html", pageContent);
        Assert.Contains("</html>", pageContent);
    }
}











